### DISCUSSION TITLE
Automatic monthly calibration

### DISCUSSION THREAD

**Discussion by balupton** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#715] [Ideas]
I use my macbook pro plugged in 99 days out of a hundred. Perhaps AlDente could automatically run the calibration every 2 weeks, so I don't have to worry about it. From what I can understand, for my case, having the charge limit set to 55% with sailing mode difference at 10% is optimal. Here's my full settings:
<img width="544" alt="CleanShot 2022-07-09 at 21 49 16@2x" src="https://user-images.githubusercontent.com/61148/178108660-6b5306bf-2410-4825-be9d-e40f7c2d106e.png">

**Comment by eighteentee** [comment]
I have been wondering this myself and it makes absolute sense to do so. Right now, I use a calendar event to remind me to do it, but an automatic method would be fabulous

**Comment by berters** [comment]
Yes please—automatic makes sense!

**Reply by eighteentee** [reply]
Is the app scriptable? Maybe a recurring script could action a full calibration?

**Reply by MatthiasKerbl** [reply]
Hi @eighteentee,

Currently, AlDente is not scriptable but we might add it in the future. However, we added support to Apple's Shortcuts with our latest version AlDente 1.19.
You can read more about it on our blog here: https://apphousekitchen.com/feature-explanation-shortcuts/

Cheers,
Matthias

**Reply by berters** [reply]
Do you think this is the best way to do it then—with a scheduled Automation in Shortcuts?

**Reply by MatthiasKerbl** [reply]
I don't know if it is the best way but it might be a possible solution right now.

**Reply by eighteentee** [reply]
I'm wondering if a Shortcut can be automatically executed at specific times like an Apple Script can? I've limited experience with Shortcuts

**Reply by berters** [reply]
@eighteentee Looks like it's possible:
https://medium.com/@richardmoult75/how-to-schedule-macos-shortcuts-96774f6972ac

Although I'd love a built-in schedule @MatthiasKerbl 😉

**Reply by MatthiasKerbl** [reply]
Hi @berters,

Thanks for the article, it looks interesting.
It is definitely on our list for potential future updates. Unfortunately, I can't tell you when we will start working on it or releasing it.

Cheers,
Matthias

**Comment by eighteentee** [comment]
I don't know if the app is scriptable though? How would one find out?

**Comment by eighteentee** [comment]
This looks promising.

**Comment by MatthiasKerbl** [comment]
Hi everyone,

We have just released AlDente 1.21.2 which has a new beta feature called Schedule. This feature allows users to set specific Actions in AlDente to run automatically on predetermined days, times, and intervals. For example, Calibration Mode can be set to run every two weeks.
You can read more about the Schedule feature and its current limitations here: https://apphousekitchen.com/feature-explanation-schedule/
You can find a full list of the Release Notes here: https://apphousekitchen.com/changelog/

Cheers,
Matthias
